packages/sitecore-jss/src/site/graphql-redirects-service.ts

```diff
@@ -48,6 +50,12 @@ export type GraphQLRedirectsServiceConfig = {
    * Override fetch method. Uses 'GraphQLRequestClient' default otherwise.
    */
   fetch?: typeof fetch;
+  /**
+   * Cache settings
+   * Specified the cache timeout and whether cache is enabled
+   * If not provided cache will be enabled with timeout of 10 seconds
+   */
+  cacheSettings?: CacheOptions;
```

Comment:
Let's extend the `GraphQLRedirectsServiceConfig` directly with this instead of a sub-prop. See [this PR](https://github.com/Sitecore/jss/pull/1218/files#diff-faa0d99d1707172962aeaa11ad2a1caa2889707c652670d89270bbc994b1573d) for example.